src/lib.rs

```diff
@@ -167,9 +201,12 @@ impl ApplicationBuilder {
         
         let general = config.general.unwrap_or_default();
 
-        let init_path = get_or(&general, "init", "init.lua");
+        let init_path = match self.command {
+            Some(ref command) => command.clone().into_os_string().into_string().unwrap(),
```

Comment:
May want to be careful when doing an unwrap since `into_os_string().into_string()` may return a string or the original `OsString`, which is not always compatible with functions. You can leave it as is for now since I will be working on something to handle errors properly

Reply:
Well, `init_path` should be the `Path`/`PathBuf` anyway.